pkg/machines/vnc.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <!--
+        noVNC wrapper based on
+            - https://github.com/novnc/noVNC/blob/master/vnc_auto.html
+            - cockpit/examples/poc-vnc
+
+    Connect parameters are provided in query string:
+        http://example.com/?host=HOST&port=PORT&encrypt=1&true_color=1
+    or the fragment:
+        http://example.com/#host=HOST&port=PORT&encrypt=1&true_color=1
+    -->
+    <title>noVNC</title>
+
+    <meta charset="utf-8">
+
+    <!-- Always force latest IE rendering engine (even in intranet) & Chrome Frame
+                Remove this if you use the .htaccess -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+
+    <!-- Apple iOS Safari settings -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+    <!-- App Start Icon  -->
+    <link rel="apple-touch-startup-image" href="images/screen_320x460.png" />
+    <!-- For iOS devices set the icon to use if user bookmarks app on their homescreen -->
+    <link rel="apple-touch-icon" href="images/screen_57x57.png">
+
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="include/base.css" title="plain">
+
+    <script src="include/util.js"></script>
+    <script src="../base1/jquery.js"></script>
+    <script src="../base1/cockpit.js"></script>
+
+    <style>
+        .vnc-main-menu {
+            background-color: white;
+            padding-bottom: 5px;
+            font-size: 13px;
+        }
+
+        .vnc-button {
+            background-color: #e7e7e7;
+            color: black;
+            border: none;
+            padding: 5px 15px;
+            text-align: center;
+            text-decoration: none;
+            display: inline-block;
+            font-size: 16px;
+        }
+    </style>
```

Comment:
I believe inline <style> violates CSP? Can this move into the .less file? There are also a bunch of `style=` properties below which would have the same problem